### ISSUE TITLE
Starting time

### ISSUE THREAD

**Issue by FunnyDevs** [justasm/WindowView#4]
Is possibile to choose when to start the sensor listening (and so the panning)???

**Comment by justasm** [comment]
Not out of the box for the moment.

You could subclass WindowView as follows:

``` java
public class ManualWindowView extends WindowView {
    // .. standard constructors

    public void startTracking(){
        sensor.startTracking(getSensorSamplingPeriod());
    }

    public void stopTracking(){
        sensor.stopTracking();
    }
}
```

and then control it as such, although it is clearly not ideal as it would still be active by default.

**Comment by justasm** [comment]
Adding more options for controlling this is not difficult and I could do so in a future version. Pull requests are of course also welcome!

I can see offering the following options:
- tilt tracking starts / stops automatically (keep current behavior as default);
- tilt tracking is off by default, and you call e.g. `WindowView#startTiltTracking()` to start (or stop) at your leisure;
- tilt tracking is off by default, and you call e.g. `WindowView#startTiltTracking(TiltSensor)` to start (or stop) at your leisure, supplying a `TiltSensor` that you manage manually; this would allow you to optimize if using many `WindowView`s by using only one `TiltSensor`.

Do you agree that these options would cover the majority of use cases, or am I missing something?

**Comment by FunnyDevs** [comment]
Yes, i think these option cover a lot of use case! The second options are useful when user has to add some animations to the windowsview and prefers to start tilting after animation ends. 
The second instead, as you said, to use multiple windowview

**Comment by justasm** [comment]
The aforementioned options are now available in the 0.2.0 release.

Refer to [TiltSensorMode](https://github.com/justasm/WindowView/blob/733d5df6d3c4fb0e87d62b8d314e05e1026bb40f/library/src/main/java/com/jmedeisis/windowview/WindowView.java#L88) or the documentation for `wwv_tilt_sensor_mode` in the [README](https://github.com/justasm/WindowView/blob/733d5df6d3c4fb0e87d62b8d314e05e1026bb40f/README.md) for details.
